src/pages/Vote/DelegateModal/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Formik } from 'formik';
+import { Typography } from '@mui/material';
+import { Modal, NoticeInfo, FormikSubmitButton, FormikTextField, TextButton } from 'components';
+import Path from 'constants/path';
+import { useTranslation } from 'translation';
+import addressValidationSchema from './addressValidationSchema';
+import { useStyles } from './styles';
+
+interface IDelegateModalProps {
+  onClose: () => void;
+  open: boolean;
+  address: string;
+}
+
+const DelegateModal: React.FC<IDelegateModalProps> = ({ onClose, open, address }) => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+  const handleSubmit = () => {};
+  return (
+    <Modal isOpened={open} handleClose={onClose} title={t('vote.delegateVoting')}>
+      <>
+        <NoticeInfo
+          title={t('vote.pleasePayAttention')}
+          description={t('vote.youCanDelegateVotes')}
+        />
+        <Formik
+          initialValues={{
+            address: '',
+          }}
+          onSubmit={handleSubmit}
+          validationSchema={addressValidationSchema}
+          validateOnMount
+          validateOnChange
+        >
+          {({ setFieldValue }) => (
+            <>
+              <div css={styles.inputLabels}>
+                <Typography>{t('vote.delegateAddress')}</Typography>
+                <TextButton css={styles.inline} onClick={() => setFieldValue('address', address)}>
+                  {t('vote.pasteYourAddress')}
+                </TextButton>
+              </div>
+              <FormikTextField placeholder={t('vote.enterContactAddress')} name="address" />
```

Comment:
Nit: we can add a max length of 42 characters to this field since that's the length of addresses.